apps/AgeDetection/agedetection.cpp

```diff
@@ -97,12 +97,13 @@ class AgeDetector {
 
         auto blockingTimeStart = std::chrono::system_clock::now();
         output->pushData(resultBlock, outputThreshold);
-        NDN_LOG_INFO("Output Queue Size: " << output->size());
+        NDN_LOG_DEBUG("Output Queue Size: " << output->size());
         auto blockingTimeEnd = std::chrono::system_clock::now();
 
         std::chrono::duration<double> elapsedBlockingTime =
             (blockingTimeEnd - blockingTimeStart);
-        NDN_LOG_INFO("Blocking Time: " << elapsedBlockingTime.count());
+        //        NDN_LOG_INFO("Blocking Time: " <<
+        //        elapsedBlockingTime.count());
```

Comment:
Turn into a debug statement?

Reply:
I am changing the outputs like this: IceFlow related outputs are DEBUG outputs and application specific outputs are INFO outputs.